app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/EISConnector.scala

```diff
@@ -104,6 +112,20 @@ class EISConnectorImpl @Inject() (
       .executeAndDeserialise[CreateRecordResponse]
   }
 
+  override def removeRecord(
```

Comment:
we should start to put these request into their own file. E.g. RemoveEisConnector for Remove, CeateEISConnector for Create (this maybe can be done in a different time as create was done on a different ticket), etc. This will make test more manageable and file smaller and more readable.

Reply:
we could do that but if we see the code in it, is all calling with just endpoint so not sure if we need to create connectors specific to actions?